projects/index.md

```diff
@@ -61,6 +61,8 @@ comments: false
 ### Documentation
 
 * [grape-swagger](https://github.com/ruby-grape/grape-swagger): Provides an autogenerated documentation for your Grape API.
+* [grape-swagger-entity](https://github.com/ruby-grape/grape-swagger-entity): Provides grape-entity models parsing for swagger autogenerated documentation
```

Comment:
Both missing periods at the end of the sentence. From there hell breaks loose. Please `--amend` ;)